lxc/storage_volume.go

```diff
@@ -1807,7 +1807,10 @@ func (c *cmdStorageVolumeShow) Command() *cobra.Command {
     Will show the properties of a custom volume called "data" in the "default" pool.
 
 lxc storage volume show default container/data
-    Will show the properties of the filesystem for a container called "data" in the "default" pool.`))
+    Will show the properties of the filesystem for a container called "data" in the "default" pool.
+
+lxc storage volume show default image/6b15fb...dafea3
```

Comment:
Lets not use "..." as that isn't what the command accepts and could confuse.

Reply:
Maybe a shorter id to get the point, or the full one?

Reply:
or use a custom volume example

Reply:
The custom volume example is already there, so if we put a comment telling when to provide a type, this extra example is probably obsolete.